core-android/src/main/java/com/mux/stats/sdk/muxstats/MuxUiDelegate.kt

```diff
@@ -12,9 +12,10 @@ import com.mux.stats.sdk.core.util.MuxLogger
 
 /**
  * Allows implementers to supply data about the view and screen being used for playback
+ * @param PV An object that can be used to access view and screen metrics, like a [View]
```

Comment:
Was PV intentionally added here?

Reply:
I'm just documenting a type param that was already present. It used to be `<PlayerView>` but I was worried that would be confusing. It might lead people to believe there's some specific `View` (ie, implementation of `android.view.View`) that this works with, which is not the case.

These generics are necessary since different players have different "Player View"s, and we also have to consider use-cases audio-only and apps that use Compose; in these cases there wouldn't be a `android.view.View` at all.